platform/src/common/components/Collocation/DeviceStatus/Table/DataTable.jsx

```diff
@@ -12,7 +12,7 @@ import { useGetCollocationResultsQuery } from '@/lib/store/services/collocation'
 import Dropdown from '@/components/Dropdowns/Dropdown';
 import InfoIcon from '@/icons/Common/info_circle.svg';
 import Modal from '@/components/Modal/Modal';
-import axios from 'axios';
+import createAxiosInstance from '../../../../../core/apis/axiosConfig';
```

Comment:
Possible to have an absolute path here? And avoid these endless path chains? @OchiengPaul442